### ISSUE TITLE
Uses unhandled promise rejections

### ISSUE THREAD

**Issue by haf** [n8n-io/n8n#1269]
**Describe the bug**

During startup, if there's a promise rejection, it's not caught by n8n

```
n8n-759f56f8d7-c6qkm n8n UserSettings got generated and saved to: /home/node/.n8n/config
n8n-759f56f8d7-c6qkm n8n (node:7) UnhandledPromiseRejectionWarning: Error: self signed certificate in certificate chain
n8n-759f56f8d7-c6qkm n8n     at TLSSocket.onConnectSecure (_tls_wrap.js:1502:34)
n8n-759f56f8d7-c6qkm n8n     at TLSSocket.emit (events.js:314:20)
n8n-759f56f8d7-c6qkm n8n     at TLSSocket._finishInit (_tls_wrap.js:937:8)
n8n-759f56f8d7-c6qkm n8n     at TLSWrap.ssl.onhandshakedone (_tls_wrap.js:711:12)
n8n-759f56f8d7-c6qkm n8n (node:7) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
n8n-759f56f8d7-c6qkm n8n (node:7) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
n8n-759f56f8d7-c6qkm n8n (node:7) PromiseRejectionHandledWarning: Promise rejection was handled asynchronously (rejection id: 1)
n8n-759f56f8d7-c6qkm n8n (node:7) UnhandledPromiseRejectionWarning: Error: There was an error: self signed certificate in certificate chain
n8n-759f56f8d7-c6qkm n8n     at Object.error (/usr/local/lib/node_modules/n8n/node_modules/@oclif/errors/lib/index.js:26:15)
n8n-759f56f8d7-c6qkm n8n     at Start.error (/usr/local/lib/node_modules/n8n/node_modules/@oclif/command/lib/command.js:60:23)
n8n-759f56f8d7-c6qkm n8n     at /usr/local/lib/node_modules/n8n/dist/commands/start.js:134:22
n8n-759f56f8d7-c6qkm n8n (node:7) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
```

**To Reproduce**
1. Configure a self-signed certificate, without setting unauthorized to true

**Expected behavior**

```js
(() => {
  // start n8n
 }).then(() => {}, error => console.error(error))`
```

**Environment (please complete the following information):**
 - n8n Version [e.g. 0.26.0] 0.98.0

**Comment by janober** [comment]
Thanks a lot for reporting this. Got fixed and will be released with the next version.

**Comment by janober** [comment]
Got released with n8n@0.99.0
